packages/mobile/App.tsx

```diff
@@ -1,18 +1,35 @@
 import React from 'react';
 import CreateWorkout from '@mobile/features/configure_workout/CreateWorkoutScreen';
+import ExerciseList from '@mobile/features/configure_workout/ExerciseList';
 import Index from '@mobile/features/index/IndexScreen';
 import { RootStackParamList } from '@mobile/navigation/NavigationConstants';
+import useBoundStore from '@mobile/stores/useBoundStore';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
 const App = (): React.JSX.Element => {
   const RootStack = createNativeStackNavigator<RootStackParamList>();
+  const currentWorkout = useBoundStore(state => state.currentWorkout);
 
   return (
     <NavigationContainer>
       <RootStack.Navigator initialRouteName="Index">
-        <RootStack.Screen name={'Index'} component={Index} />
+        <RootStack.Screen
+          name={'Index'}
+          component={Index}
+          options={{
+            title: currentWorkout?.name,
```

Comment:
hmm i think we're not on the same page about what this index page is, it shouldn't have a current workout on it, it's more or less a debug page to let us jump to different components while we build them out and we can eventually replace it with a proper home page type of thing

Reply:
Active workout was created to test zustand support and it didn't make sense to put it anywhere else for the time being. I agree that there should be a proper home page but this was strictly for testing purposes.